templates/Collector/db.html.twig

```diff
@@ -100,8 +107,8 @@
         .time-container { position: relative; }
         .time-container .nowrap { position: relative; z-index: 1; text-shadow: 0 0 2px #fff; }
         .time-bar { display: block; position: absolute; top: 0; left: 0; bottom: 0; background: #e0e0e0; }
-        .sql-runnable.sf-toggle-content.sf-toggle-visible { display: flex; flex-direction: column; }
-        .sql-runnable button { align-self: end; }
+        .sql-runnable.sf-toggle-content.sf-toggle-visible { display: flex; flex-direction: row; align-items: center}
+        .sql-runnable pre { flex-grow: 1}
```

Comment:
I'm not a fan of this design change. Let's have buttons next to each other, instead on top of each other